IPython/core/shellapp.py

```diff
@@ -100,7 +100,7 @@
 
 class InteractiveShellApp(Configurable):
     """A Mixin for applications that start InteractiveShell instances.
-    
+
```

Comment:
Not a big deal, but if you're contributing to IPython/Jupyter more in the future, we will ask you to configure your editor not to strip whitespace in lines you haven't touched - it can create needless merge conflicts, and it makes `git blame` less useful.